src/sentry_os.c

```diff
@@ -195,8 +195,105 @@ sentry__get_os_context(void)
 }
 #elif defined(SENTRY_PLATFORM_UNIX)
 
+#    include <fcntl.h>
 #    include <sys/utsname.h>
 
+#    if defined(SENTRY_PLATFORM_LINUX)
+#        define OS_RELEASE_MAX_LINE_SIZE 128
+#        define OS_RELEASE_MAX_KEY_SIZE 64
+#        define OS_RELEASE_MAX_VALUE_SIZE 128
+
+static int
+parse_os_release_line(const char *line, char *key, char *value)
+{
+    const char *equals = strchr(line, '=');
+    if (equals == NULL)
+        return 1;
+
+    unsigned long key_length = equals - line;
+    strncpy(key, line, key_length);
+    key[key_length] = 0;
+
+    unsigned long value_length = strlen(equals + 1);
+    strncpy(value, equals + 1, value_length);
+    value[value_length] = 0;
+
+    // some values are wrapped in double quotes
+    if (value[0] == '\"') {
+        value[value_length - 1] = 0;
+        memmove(value, value + 1, value_length);
```

Comment:
Shouldn't this be `value_length-2`? As we want to ignore the quotation marks at beginning + end?

Reply:
instead of operating on `value` and using a `memmove`, it might be a good idea to detect this before calling into `strncpy`, and adjust the slice range then.